tests/diffusion_2d_p0_adaptive_parallel/diffusion_2d_p0_adaptive_parallel.xml

```diff
@@ -1,51 +1,32 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I was wondering what the hell is going on with this file. But it looks like this was a hand-crafted .xml originally and this is what happens if you open and save it in diamond.